mm2src/lp_ordermatch.rs

```diff
@@ -2922,7 +3353,7 @@ pub async fn orderbook(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, Strin
     let rel_coin = try_s!(rel_coin.ok_or("Rel coin is not found or inactive"));
     let base_coin = try_s!(lp_coinfindᵃ(&ctx, &req.base).await);
     let base_coin: MmCoinEnum = try_s!(base_coin.ok_or("Base coin is not found or inactive"));
-    let request_orderbook = true;
+    let request_orderbook = false;
```

Comment:
Just a reminder to set this to true after my side is done

Reply:
Yeah, I'm going to add a TODO here.